Automattic/wp-calypso
# Performance Profiler: Use an accordion for small screens in CrUX section

Resolves https://github.com/Automattic/dotcom-forge/issues/8911

## Proposed Changes

* Extract the `CoreWebVitalsDetails` to its own component so it can be reused in both, big and small (i.e. `Mobile`) screens
* Use `FoldableCard` component for small screens

https://github.com/user-attachments/assets/d47dd685-c936-4570-8b15-5529c867544f


## Why are these changes being made?


* For small screens, the current tab component do not fit so, foldable cards have been used according to the design

## Testing Instructions



* Navigate to the Speed Test tool with a result
* Resize the screen and check that the CrUX section changes from a Tab component to an Accordion component for small screens
* Check the behavior of the Accordion component works as expected
* Check the style of the headers of the Accordion matches Figma Iix1FFHIQdfECIXn5JKT5s-fi-4356_19874
* Check this related PR still works as expected https://github.com/Automattic/wp-calypso/pull/93997

**Note**: The responsive functionality of the content of the card will be handled as part of https://github.com/Automattic/wp-calypso/pull/94031

## Pre-merge Checklist
